recipes-domx/meta-xt-prod-domx/conf/layer.conf

```diff
@@ -0,0 +1,8 @@
+# We have a conf and classes directory, add to BBPATH
+BBPATH =. "${LAYERDIR}:"
+
+BBFILE_COLLECTIONS += "meta-xt-prod-domx"
+BBFILE_PATTERN_meta-xt-prod-domx = "^${LAYERDIR}/"
+BBFILE_PRIORITY_meta-xt-prod-domx = "7"
```

Comment:
Usually we use PRIORITY 6 for our layers.
I have nothing against 7, just side note.
But, if there are some reasons to have higher priority - may be it's good to mention here.

Reply:
The recipes in meta-xt-images have priority 6. 
The recipes in meta-xt-prod-devel  extend recipes in meta-xt-image and would  better to have higher priority. 
But because meta-xt-prod-domx does not contain any recipe, I does not matter for now.